lib/internal/blob.js

```diff
@@ -89,16 +123,17 @@ function getSource(source, encoding) {
   return [source.byteLength, source];
 }
 
-class InternalBlob extends JSTransferable {
-  constructor(handle, length, type = '') {
-    super();
-    this[kHandle] = handle;
-    this[kType] = type;
-    this[kLength] = length;
-  }
-}
-
-class Blob extends JSTransferable {
+class Blob {
+  /**
+   * @typedef {string|ArrayBuffer|ArrayBufferView|Blob} SourcePart
+   *
+   * @param {SourcePart[]} [sources]
+   * @param {{
+   *   endings? : string,
+   *   type? : string,
+   * }} options
```

Comment:
```suggestion
   * }} [options]
```
The `options` parameter is optional.